insights/combiners/grubby.py

```diff
@@ -0,0 +1,38 @@
+"""
+Grubby
+======
+
+Combiner for command ``/usr/sbin/grubby`` parsers.
+
+This combiner uses the parsers:
+:class:`insights.parsers.grubby.GrubbyDefaultIndex`,
+:class:`insights.parsers.grubby.GrubbyInfoAll`.
+"""
+
+from insights.core.exceptions import ParseException
+from insights.core.plugins import combiner
+from insights.parsers.grubby import GrubbyDefaultIndex, GrubbyInfoAll
+
+
+@combiner(GrubbyInfoAll, GrubbyDefaultIndex)
+class Grubby(object):
+    """
+    Combine command "grubby" parsers into one Combiner.
+
+    Attributes:
+        boot_entries (dict): All boot entries indexed by the entry "index"
+        default_index (int): the numeric index of the current default boot entry
```

Comment:
As a combiner, it would be good to combine more useful attribute here, e.g. devs can use the combiner rather than access each single parser to fetch particular info.